package.json

```diff
@@ -64,6 +64,7 @@
     "dev-novel": "0.0.4",
     "doctoc": "1.3.0",
     "enzyme": "2.9.1",
+    "enzyme-to-json": "2.0.1",
```

Comment:
Can you explain why did you need this new dependency?

Reply:
This dependency allow to pass the Enzyme wrapper to the `expect` function to be able to directly use snapshot on it.

```js
import { shallow } from 'enzyme';
import serializer from 'enzyme-to-json/serializer';

expect.addSnapshotSerializer(serializer);

const component = shallow(<div />);

expect(component).toMatchSnapshot();
```